documentation/modules/exploit/windows/browser/ms14_064_ole_code_execution.md

```diff
@@ -0,0 +1,94 @@
+## Vulnerable Application
+
+This module exploits the Windows OLE Automation array vulnerability, [CVE-2014-6332](https://www.cvedetails.com/cve/cve-2014-6332). The vulnerability is known to affect Internet Explorer 3.0 until version 11 within Windows 95 up to Windows 10, and no patch for Windows XP. However, this exploit will only target Windows XP and Windows 7 box due to the Powershell limitation. Windows XP by defaults supports VBS, therefore it is used as the attack vector. On other newer Windows systems, the exploit will try using Powershell instead.
+
+## Verification Steps
+
+1. Install the application
+1. Start msfconsole
+1. Do: ```use exploit/windows/browser/ms14_064_ole_code_execution```
+1. Do: ```Choose a payload and set any specific options```
+1. Do: ```run```, after a target browses to the generated URL, you should receive a session.
+
+## Options
+
+**TRYUAC**
+
+Ask victim to start as Administrator. This option only works on Windows 7 targets.
+
+**AllowPowershellPrompt**
+
+Allow exploit to try Powershell. If exploiting a Windows 7 target you need to enable this option.
+
+## Scenarios
+
+Example usage against a Windows XP Professional SP3 x86 running Internet Explorer 8.0.6001.18702. The attacker's IP is `192.168.2.3` and the victim's IP is `192.168.2.207`. 
```

Comment:
Lets do a sub header here:
### Windows XP Pro SP3 x86 with IE 8.0.6001.18702
(`### Windows XP Pro SP3 x86 with IE 8.0.6001.18702`)

Then you can remove the first sentence and just leave the second